### PR TITLE
Added output file extension option

### LINKED ISSUE #48: Choose Output File Extension
Would be super helpful if there was an option to choose an output file extension other than html. I am creating handlebars files from mjml and VSCode treats them differently when the file extension is html instead of hbs

### PR DESCRIPTION
**Pull request by sinklair**
Fixes #48

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -47,7 +51,7 @@ module.exports = function mjml(mjmlEngine, options) {
 
       // [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues
       output.contents = Buffer.from(render.html);
-      output.path = replaceExt(file.path.toString(), ".html");
+      output.path = replaceExt(file.path.toString(), localOptions.fileExt);
```
Comment (kmcb777) [inline src/index.js]: maybe just `localOptions.fileExt || '.html'` here, instead of the `if` before

**Comment by kmcb777** [comment]
nice idea 👍

**Comment by kmcb777** [comment]
maybe we should add this option directly in mjml though

**Comment by iRyusa** [comment]
@kmcb777 MJML only output string, so there's no file involved (that's why there's a filePath for mj-include)

File: src/index.js (hunk 1 above)
Comment (sinklair) [reply src/index.js]: Updated

**Comment by kmcb777** [comment]
@iRyusa ah yes indeed, but we could add it to the cli

then we can merge this one anyway :)

File: src/index.js (hunk 1 above)
Comment (kmcb777) [reply src/index.js]: 👍

**Pull request merged**